services/webhook/tests/test_llm_client.py

```
@@ -379,6 +379,46 @@ def capture(url, *, json, headers, timeout):
+    """Grug's own review ensemble (coder + reasoner arms) carries
+    X-Spark-Priority: interactive so the spark-gateway priority queue
+    (githumps/infra#1768) lets it jump ahead of Hermes's long agentic turns
+    on the same shared, single-generation-slot Ollama target - the exact
+    2026-07-12 incident this header exists to prevent. Deep depth so BOTH
+    arms fire (see test_deep_review_consults_both_arms_and_merges_findings)
+    - a coder-only run would pass even if the reasoner arm's config lost
+    the header."""
+    monkeypatch.setenv("GRUG_REVIEW_DEPTH", "deep")
+    captured: list = []
+
+    def capture(_url, *, headers, **_kwargs) -> httpx.Response:
```

Ruff (0.15.20)

[warning] Missing type annotation for `**_kwargs`

(ANN003)